keyboards/ivy/rev1/rev1.c

```diff
@@ -0,0 +1,14 @@
+#include "ivy.h"
+
+void matrix_init_kb(void) {
+	
+	// JTAG disable for PORT F. write JTD bit twice within four cycles.
+    MCUCR |= (1<<JTD);
```

Comment:
This isn't actually needed anymore.  This is being done in the core code for, so it is redundant. 

https://github.com/qmk/qmk_firmware/blob/master/tmk_core/common/keyboard.c#L132-L138